app/views/label_formats/_form.html.haml

```diff
@@ -0,0 +1,24 @@
+-#
+  Copyright (c) 2021, Katholische Landjugendbewegung Paderborn. This file is part of
+  hitobito_kljb and licensed under the Affero General Public License version 3
+  or later. See the COPYING file at the top-level directory or at
+  https://github.com/hitobito/hitobito_kljb.
+
+= entry_form(buttons_bottom: false) do |f|
+  = hidden_field_tag :global, params[:global]
+
+  #main.row-fluid
+    %article.span6
+      = f.labeled_input_fields :name
+      = f.labeled_collection_select(:page_size, LabelFormat.available_page_sizes, :to_s, :to_s, {}, class: 'span4')
+      = f.labeled(:landscape) do
+        = f.inline_radio_button(:landscape, false, t('.portrait'))
+        = f.inline_radio_button(:landscape, true, t('.landscape'))
+      = f.labeled_input_fields :font_size, :count_horizontal, :count_vertical
+      = f.labeled_input_field :width, help_inline: 'mm'
+      = f.labeled_input_field :height, help_inline: 'mm'
+      = f.labeled_input_field :padding_top, help_inline: 'mm'
+      = f.labeled_input_field :padding_left, help_inline: 'mm'
+      = f.labeled_input_field :nickname if Person::PUBLIC_ATTRS.include?(:nickname)
```

Comment:
Das wir das jetzt  komplett überschrieben kann diese Zeile hier auch gleich raus. Das wurde für KLJB im Core angepasst, sie wollen den Nickname nicht verwenden.

Ich hatte gehofft, ohne auszukommen, aber vermutlich ist das besser so...

Reply:
Hmm ja ausser man macht es abhängig von dem ersten entry in den `Settings.countries.prioritized` aber das fühlt sich für mich auch nicht ganz korrekt an...